### DISCUSSION TITLE
Добавить свойство, которое бы ограничивало просмотр месяцев и годов в компоненте Calendar.

### DISCUSSION THREAD

**Discussion by Rezetchik** [VKCOM/VKUI#6565] [Ideas] [answered]
Добавить свойство, которое бы ограничивало просмотр месяцев и годов, которые не входят в [minDateTime, maxDateTime]. Cейчас, при ограничении, компонент не скрывает лишние года, всё так же в полосе прокрутки остаётся до 1124 года.
![DPNebhlQT1Q](https://github.com/VKCOM/VKUI/assets/146764865/9a6a6340-ae98-4bb0-835f-d0d9c99c928e)

**Comment by Semigradsky** [comment]
Текущее поведение выглядит как баг

**Comment by mendrew** [comment, marked as answer]
Сделали задачу. https://github.com/VKCOM/VKUI/issues/6936

Обсудили с дизайнерами и пришли к выводу что действительно не следует давать возможность выбора года/месяца, если они не входят в разрешенный диапазон. Полностью убирарть опции из селектов не будем, но будем нашеивать на них `disabled`.
